cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=100)
+    duration = serializers.IntegerField(required=True)
+    description = serializers.CharField(required=False, allow_blank=True, max_length=500)
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("title", instance.title)
+        instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
```

Comment:
The `update` method does not update the `description` field. Add logic to update `instance.description` using `validated_data.get('description', instance.description)` to ensure all fields are updated correctly.